src/lib.rs

```diff
@@ -589,6 +733,114 @@ impl Contract {
         self.edit_proposal_internal(id, body.into(), proposal.snapshot.labels)
     }
 
+    #[payable]
+    pub fn edit_proposal_linked_rfp(&mut self, id: ProposalId, rfp_id: Option<RFPId>) -> Promise {
+        near_sdk::log!("edit_proposal_linked_rfp");
```

Comment:
I don't think there is any value in these logs, do you? I know that they were added before, but I don't want us to keep copying something useless going forward.